pom.xml

```diff
@@ -86,7 +86,7 @@
     <commons-lang.version>3.17.0</commons-lang.version>
     <jackson.version>2.17.2</jackson.version>
     <javaparser.version>3.26.2</javaparser.version>
-    <jgit.version>6.10.0.202406032230-r</jgit.version>
+    <zip4j.version>2.11.5</zip4j.version>
```

Comment:
What features is `java.util.zip.ZipFile` missing?

Reply:
None afaik, it was just much more convenient to use a higher level library (less LOCs, maintenance etc.).

Reply:
@electrum the dependency is not exposed to the end user and only used internally in Maven plugins, do you think we should go the extra mile to remove it?

Reply:
Sorry, I should have clarified that this was just a question and not a blocking concern about the PR.